README.md

```diff
@@ -12,186 +12,277 @@ the MIT License – please take the following message to heart:*
 > LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 > OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 > SOFTWARE.
-*
 
 ## Features
+
 * Easy programmatic trading.
 * A customizable, websocket-synced Order Book implementation.
 * API clients with convenient methods for every API endpoint.
-* Abstracted interfaces – don't worry about HMAC signing or JSON formatting,
-  the library does it for you.
+* Abstracted interfaces – don't worry about HMAC signing or JSON formatting, the
+  library does it for you.
 * Semantic versioning.
 
 ## Installation
+
 ```bash
-# From NPM
+# from npm
 npm install gdax
-# From Github
+
+# from GitHub
 npm install coinbase/gdax-node
 ```
 
+You can learn about the API responses of each endpoint [by reading our
+documentation](https://docs.gdax.com/#market-data).
+
 ## Quick Start
 
-### The Public API Client
-The GDAX API has both public and private endpoints. If you're only
-interested in the public endpoints, you should use a `PublicClient`.
+The GDAX API has both public and private endpoints. If you're only interested in
+the public endpoints, you should use a `PublicClient`.
 
-```javascript
-var Gdax = require('gdax');
-var publicClient = new Gdax.PublicClient(productID*, endpoint*);
+```js
+const Gdax = require('gdax');
+const publicClient = new Gdax.PublicClient();
 ```
 
-- `productID` is OPTIONAL: defaults to 'BTC-USD' if not specified.
-- `endpoint` is OPTIONAL: defaults to https://api.gdax.com if not specified.
+All API methods, unless otherwise specified, can be used in a manner consistent
+with both promises or callbacks.
 
-All API methods are callback based. Your callback should accept three arguments:
+### Using Promises
 
+```js
+publicClient
+  .getProducts()
+  .then(data => {
+    // work with data
+  })
+  .catch(error => {
+    // handle the error
+  });
 ```
-var callback = function(err, response, data) {
-  // your code here.
-};
+
```

Comment:
Perhaps add some description here: "Promises can be used with callbacks, as well as using `async/await` in modern environments." (or something similar).